pkgs/tools/games/slipstream/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib, fetchFromGitHub, stdenv, makeWrapper, buildMaven, maven, jdk8 }:
+let
+  settings = (buildMaven ./project-info.json).settings;
+  jdk = jdk8;
+in
+stdenv.mkDerivation rec {
```

Comment:
I think we should use the `finalAttrs` pattern here.

Since this is your first PR, let me share a couple of useful links regarding this:

1. History: https://discourse.nixos.org/t/avoid-rec-expresions-in-nixpkgs/8293
2. Documentation: https://nixos.org/manual/nixpkgs/unstable/#mkderivation-recursive-attributes
3. Recent example of implementation: https://github.com/NixOS/nixpkgs/compare/17f96f7b978e61576cfe16136eb418f74fab9952..9e6ea843e473d34d4f379b8b0d8ef0425a06defe

Reply:
Ah, thank you! The finalAttrs pattern is a million times better! :) I've struggled with overriding the source version of other packages before so am very happy to implement this pattern - and thank you for the feedback!